src/basis/data.js

```diff
@@ -2594,7 +2603,15 @@
       flushAllDataset();
     }
 
-    return function(state){
+    Dataset.preventAccumulations = function(dataset){
+      if (setStateCount) {
```

Comment:
It's not correct. Since accumulate state can be turned after `preventAccumulations` call. Better to check value in `eventCache`.

```js
if (eventCache[dataset.basisObjectId] !== PREVENT_ACCUMULATIONS)
{
  ...
}
```

But I also worry about `setAccumulateState` true/false after `preventAccumulations` call. It potentially could break prevention.